src/plugins/tagfilter/tagfilter.js

```diff
@@ -261,6 +271,25 @@ $document.on( "change", selectorCtrl, function( event )  {
 	$elm.trigger( "wb-contentupdated" );
 } );
 
+// Show no result message if on Firefox -- Remove once Firefox supports ":has()"
+$document.on( "wb-contentupdated", selector, function()  {
+	let supportsHas = getComputedStyle( document.documentElement ).getPropertyValue( "--supports-has" );
```

Comment:
Considering the closure scope currently defined for this plugin, this call should be made from the `window` variable instead of calling the global directly.

```suggestion
	let supportsHas = window.getComputedStyle( document.documentElement ).getPropertyValue( "--supports-has" );
```